compiler/rustc_error_codes/src/error_codes/E0540.md

```diff
@@ -0,0 +1,26 @@
+An attribute was given an invalid number of arguments
+
+Erroneous code example:
+
+```compile_fail,E0540
+#[inline()] // error! should either have a single argument, or no parentheses
```

Comment:
new generic error code for argument mismatch. 540 was not used.

Reply:
Likely removed. Please use the new biggest number instead.